### DISCUSSION TITLE
made the packages for Slackware and how to start budgie-desktop

### DISCUSSION THREAD

**Discussion by adcdam** [BuddiesOfBudgie/budgie-desktop#167] [Q&A]
Hi, i made the packages for budgie-desktop but something must be wrong, it take a ton of time to start, i was using tbsm and to be able to launch budgie-desktop i did chmod +x budgie-desktop, what is the correct way to start budgie-desktop any login manager recommended?

**Comment by JoshStrobl** [comment]
LightDM is the recommended login manager. Most use it alongside slick-greeter.
